File: QuizApplication.Backend/API Contracts/QuizApplication.Contracts/Entities/Question.cs
using System.Collections.Generic;

namespace QuizApplication.Contracts.Entities
{
    public class Question : BaseEntity
    {
        public int TestId { get; set; }
        public int TimeLimit { get; set; }
        public double Points { get; set; }
        public string Description { get; set; }
        public List<QuestionAnswer> QuestionAnswers { get; set; }
        public Test Test { get; set; }
    }
}

File: QuizApplication.Client/src/app/models/question.ts
import { QuestionAnswer } from './questionAnswer';

export class Question {
    public id: number;
    public testId: number;
    public timeLimit: number;
    public points: number;
    public description: string;
    public questionAnswers: QuestionAnswer[]

    constructor() {
        this.questionAnswers = new Array();
    }
}